plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -703,7 +704,7 @@ private TableInput convertGlueTableToTableInput(com.amazonaws.services.glue.mode
                 .withPartitionKeys(glueTable.getPartitionKeys())
                 .withViewOriginalText(glueTable.getViewOriginalText())
                 .withViewExpandedText(glueTable.getViewExpandedText())
-                .withTableType(glueTable.getTableType())
+                .withTableType(getTableTypeNullable(glueTable))
```

Comment:
Are we ok with passing eventually `null` here ? 
Why not `EXTERNAL_TABLE.name()` ?

Reply:
we're passing into Glue API, and we should do whatever it takes to preserve original input
it would be going beyond this method responsibility, if we additionally applied some "business logic" here